front/pages/w/[wId]/assistant/new.tsx

```diff
@@ -154,164 +154,159 @@ export default function AssistantNew({
       navChildren={<AssistantSidebarMenu owner={owner} />}
     >
       {!conversation ? (
-        <>
-          <PageHeader
-            title="Welcome to Assistant"
-            icon={ChatBubbleBottomCenterTextIcon}
-          />
-
-          {/* GETTING STARTED */}
-          <div className="flex flex-col gap-8 pb-32">
-            <div className="mt-6 flex flex-col gap-2">
-              <div className="flex flex-row gap-2">
-                <Icon visual={FlagIcon} size="md" />
-                <span className="text-lg font-bold">Getting started?</span>
-              </div>
-              <p className="text-element-700">
-                Using Assistants is easy as asking a question to a friend or a
+        <div className="s-text-sm s-font-normal s-text-element-800">
```

Comment:
`s-` classes are not supported in `front` so I guess you can just remove that div?